packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -308,9 +318,17 @@ class Frame extends React.Component {
       return;
     }
 
+    const updateDropdownAttributes = () => {
+      if (this.state.isPositioned) {
+        this.dropdown.removeAttribute('inert');
+        this.dropdown.removeAttribute('aria-hidden');
```

Comment:
why this is being removed here ?

Reply:
We are adding the `inert` properties while mounting, all this while the dropdown will be in-accessible. Removing the inert properties will grant access to the dropdown.

If you are asking related to the `aria-hidden` property, aria-hidden will be added as a by-product when we include `inert` so it has to be removed while removing `inert` as well.

Reply:
Yeah i was asking related to aria-hidden.  thanks for the details